junction/proposals/admin.py

```diff
@@ -15,7 +15,8 @@
     ProposalSection,
     ProposalSectionReviewer,
     ProposalType,
-    ProposalVote
+    ProposalVote,
+    ProposalSectionReviewerVote
```

Comment:
I think it won't work because there is only  `up_vote` option. So, it is good to have separate model since we want to support VoteRangae say +2 to -1.